config/base_boxes.yaml

```diff
@@ -43,27 +43,59 @@ boxes:
     <<: *install_shell
     options: --scenario=katello
 
+  centos6-katello-p4-nightly:
```

Comment:
This doesn't have the common_puppet_version variable so I think this is puppet 3 while the name implies puppet 4.